ipwb/assets/webui.js

```diff
@@ -32,6 +32,9 @@ function addURIListToDOM () {
 
       li.setAttribute('data-mime', memento['mime'])
       li.setAttribute('data-status', memento['status'])
+      if ('title' in memento) {
+        li.setAttribute('title', memento['title'])
```

Comment:
If we are planning to merge #566 (after title inclusion) then this change in the `webui.js` is not desired as it will cause unnecessary conflict. Making the title data available for usage should be good enough for this PR.

Reply:
@ibnesayeed This association is an effort to get #566 merged. It currently does not deal with titles. By associating it in the UI with the `li`, we can work toward including it in the listing.

Reply:
@machawk1 #566 will be able to directly use the `title` attribute from the `memento` object. We don't need to add it to each `li` beforehand.

Reply:
@ibnesayeed Rm'd in 8b866e6. Once this PR is merged, we can prettify #566 a bit more before merging it in.